### DISCUSSION TITLE
Get prisma schema (for automatic admin interface)

### DISCUSSION THREAD

**Discussion by sehmaschine** [prisma/prisma#4425] [Q&A] [answered]
Is it possible to somehow get the prisma schema in order to generate an automatic admin interface (similar to the Django admin interface)? I guess I could read (and parse) the schema myself – but I'm interested if there's something like a json equivalent to the prisma schema which can be retrieved more easily?

**Comment by ryands17** [comment, marked as answer]
Hey @sehmaschine 👋 
Yes this is possible via he `_dmmf` property. This is highly experimental and can change with releases which is why this variable is kept private and needs to be used with caution.

```ts
const prisma = new PrismaClient()

// @ts-ignore
console.log(prisma._dmmf)
```

This will give you all the data for the models in the schema.

**Reply by sehmaschine** [reply]
Great, thanks!

**Reply by nategood** [reply]
Have there been any updates here? Is there a better-supported way to get meta data now, two years later?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
